[2025-10-31T22:10:34.510+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ingestion_pipeline.silver_dbt.dbt_run_merchants manual__2025-10-31T22:00:11.870433+00:00 [queued]>
[2025-10-31T22:10:34.531+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ingestion_pipeline.silver_dbt.dbt_run_merchants manual__2025-10-31T22:00:11.870433+00:00 [queued]>
[2025-10-31T22:10:34.533+0000] {taskinstance.py:2170} INFO - Starting attempt 2 of 2
[2025-10-31T22:10:34.565+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): silver_dbt.dbt_run_merchants> on 2025-10-31 22:00:11.870433+00:00
[2025-10-31T22:10:34.574+0000] {standard_task_runner.py:60} INFO - Started process 1139 to run task
[2025-10-31T22:10:34.589+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ingestion_pipeline', 'silver_dbt.dbt_run_merchants', 'manual__2025-10-31T22:00:11.870433+00:00', '--job-id', '21', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_dag.py', '--cfg-path', '/tmp/tmp92viw35y']
[2025-10-31T22:10:34.595+0000] {standard_task_runner.py:88} INFO - Job 21: Subtask silver_dbt.dbt_run_merchants
[2025-10-31T22:10:34.764+0000] {task_command.py:423} INFO - Running <TaskInstance: ingestion_pipeline.silver_dbt.dbt_run_merchants manual__2025-10-31T22:00:11.870433+00:00 [running]> on host 9dbfb6fb81d8
[2025-10-31T22:10:34.881+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='ingestion_pipeline' AIRFLOW_CTX_TASK_ID='silver_dbt.dbt_run_merchants' AIRFLOW_CTX_EXECUTION_DATE='2025-10-31T22:00:11.870433+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-31T22:00:11.870433+00:00'
[2025-10-31T22:10:34.885+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-31T22:10:34.890+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "docker exec -i dbt bash -lc 'cd /dbt && dbt --no-use-colors run --profiles-dir /dbt --select silver_merchants --full-refresh'"]
[2025-10-31T22:10:34.920+0000] {subprocess.py:86} INFO - Output:
[2025-10-31T22:10:36.728+0000] {subprocess.py:93} INFO - 22:10:36  Running with dbt=1.10.13
[2025-10-31T22:10:36.962+0000] {subprocess.py:93} INFO - 22:10:36  Registered adapter: clickhouse=1.9.5
[2025-10-31T22:10:37.130+0000] {subprocess.py:93} INFO - 22:10:37  Unable to do partial parsing because config vars, config profile, or config target have changed
[2025-10-31T22:10:38.636+0000] {subprocess.py:93} INFO - 22:10:38  Found 3 models, 3 sources, 488 macros
[2025-10-31T22:10:38.641+0000] {subprocess.py:93} INFO - 22:10:38
[2025-10-31T22:10:38.642+0000] {subprocess.py:93} INFO - 22:10:38  Concurrency: 4 threads (target='dev')
[2025-10-31T22:10:38.643+0000] {subprocess.py:93} INFO - 22:10:38
[2025-10-31T22:10:38.909+0000] {subprocess.py:93} INFO - 22:10:38  1 of 1 START sql table model `silver`.`silver_merchants` ....................... [RUN]
[2025-10-31T22:10:39.013+0000] {subprocess.py:93} INFO - 22:10:39  1 of 1 ERROR creating sql table model `silver`.`silver_merchants` .............. [ERROR in 0.10s]
[2025-10-31T22:10:39.023+0000] {subprocess.py:93} INFO - 22:10:39
[2025-10-31T22:10:39.024+0000] {subprocess.py:93} INFO - 22:10:39  Finished running 1 table model in 0 hours 0 minutes and 0.38 seconds (0.38s).
[2025-10-31T22:10:39.167+0000] {subprocess.py:93} INFO - 22:10:39
[2025-10-31T22:10:39.172+0000] {subprocess.py:93} INFO - 22:10:39  Completed with 1 error, 0 partial successes, and 0 warnings:
[2025-10-31T22:10:39.174+0000] {subprocess.py:93} INFO - 22:10:39
[2025-10-31T22:10:39.175+0000] {subprocess.py:93} INFO - 22:10:39  Failure in model silver_merchants (models/silver/silver_merchants.sql)
[2025-10-31T22:10:39.177+0000] {subprocess.py:93} INFO - 22:10:39    Database Error in model silver_merchants (models/silver/silver_merchants.sql)
[2025-10-31T22:10:39.178+0000] {subprocess.py:93} INFO -   Code: 60.
[2025-10-31T22:10:39.179+0000] {subprocess.py:93} INFO -   DB::Exception: Unknown table expression identifier 'bronze.merchants' in scope  src. Stack trace:
[2025-10-31T22:10:39.180+0000] {subprocess.py:93} INFO - 
[2025-10-31T22:10:39.181+0000] {subprocess.py:93} INFO -   0. DB::Exception::Exception(DB::Exception::MessageMasked&&, int, bool) @ 0x00000000137a855f
[2025-10-31T22:10:39.184+0000] {subprocess.py:93} INFO -   1. DB::Exception::Exception(String&&, int, String, bool) @ 0x000000000cae7e8e
[2025-10-31T22:10:39.185+0000] {subprocess.py:93} INFO -   2. DB::Exception::Exception(PreformattedMessage&&, int) @ 0x000000000cae7940
[2025-10-31T22:10:39.186+0000] {subprocess.py:93} INFO -   3. DB::Exception::Exception<String const&, String>(int, FormatStringHelperImpl<std::type_identity<String const&>::type, std::type_identity<String>::type>, String const&, String&&) @ 0x000000000db773ab
[2025-10-31T22:10:39.187+0000] {subprocess.py:93} INFO -   4. DB::QueryAnalyzer::resolveQuery(std::shared_ptr<DB::IQueryTreeNode> const&, DB::IdentifierResolveScope&) @ 0x0000000017830745
[2025-10-31T22:10:39.188+0000] {subprocess.py:93} INFO -   5. DB::QueryAnalyzer::resolveExpressionNode(std::shared_ptr<DB::IQueryTreeNode>&, DB::IdentifierResolveScope&, bool, bool, bool) @ 0x0000000017837221
[2025-10-31T22:10:39.189+0000] {subprocess.py:93} INFO -   6. DB::QueryAnalyzer::resolveQueryJoinTreeNode(std::shared_ptr<DB::IQueryTreeNode>&, DB::IdentifierResolveScope&, DB::QueryExpressionsAliasVisitor&) @ 0x000000001785ae58
[2025-10-31T22:10:39.189+0000] {subprocess.py:93} INFO -   7. DB::QueryAnalyzer::resolveQuery(std::shared_ptr<DB::IQueryTreeNode> const&, DB::IdentifierResolveScope&) @ 0x000000001782d021
[2025-10-31T22:10:39.190+0000] {subprocess.py:93} INFO -   8. DB::QueryAnalyzer::resolve(std::shared_ptr<DB::IQueryTreeNode>&, std::shared_ptr<DB::IQueryTreeNode> const&, std::shared_ptr<DB::Context const>) @ 0x000000001782a87a
[2025-10-31T22:10:39.190+0000] {subprocess.py:93} INFO -   9. DB::QueryAnalysisPass::run(std::shared_ptr<DB::IQueryTreeNode>&, std::shared_ptr<DB::Context const>) @ 0x0000000017829edc
[2025-10-31T22:10:39.191+0000] {subprocess.py:93} INFO -   10. DB::QueryTreePassManager::run(std::shared_ptr<DB::IQueryTreeNode>) @ 0x000000001787d276
[2025-10-31T22:10:39.191+0000] {subprocess.py:93} INFO -   11. DB::buildQueryTreeAndRunPasses(std::shared_ptr<DB::IAST> const&, DB::SelectQueryOptions const&, std::shared_ptr<DB::Context const> const&, std::shared_ptr<DB::IStorage> const&) (.llvm.8096503222789336007) @ 0x00000000183bd40d
[2025-10-31T22:10:39.192+0000] {subprocess.py:93} INFO -   12. DB::InterpreterSelectQueryAnalyzer::InterpreterSelectQueryAnalyzer(std::shared_ptr<DB::IAST> const&, std::shared_ptr<DB::Context const> const&, DB::SelectQueryOptions const&, std::vector<String, std::allocator<String>> const&) @ 0x00000000183bb32a
[2025-10-31T22:10:39.193+0000] {subprocess.py:93} INFO -   13. DB::InterpreterSelectQueryAnalyzer::getSampleBlock(std::shared_ptr<DB::IAST> const&, std::shared_ptr<DB::Context const> const&, DB::SelectQueryOptions const&) @ 0x00000000183bd86a
[2025-10-31T22:10:39.193+0000] {subprocess.py:93} INFO -   14. DB::InterpreterCreateQuery::getTablePropertiesAndNormalizeCreateQuery(DB::ASTCreateQuery&, DB::LoadingStrictnessLevel) const @ 0x00000000182e49d3
[2025-10-31T22:10:39.194+0000] {subprocess.py:93} INFO -   15. DB::InterpreterCreateQuery::createTable(DB::ASTCreateQuery&) @ 0x00000000182ed2c5
[2025-10-31T22:10:39.195+0000] {subprocess.py:93} INFO -   16. DB::InterpreterCreateQuery::execute() @ 0x0000000018303978
[2025-10-31T22:10:39.195+0000] {subprocess.py:93} INFO -   17. DB::executeQueryImpl(char const*, char const*, std::shared_ptr<DB::Context>, DB::QueryFlags, DB::QueryProcessingStage::Enum, std::unique_ptr<DB::ReadBuffer, std::default_delete<DB::ReadBuffer>>&, std::shared_ptr<DB::IAST>&, std::shared_ptr<DB::ImplicitTransactionControlExecutor>) @ 0x0000000018780440
[2025-10-31T22:10:39.196+0000] {subprocess.py:93} INFO -   18. DB::executeQuery(String const&, std::shared_ptr<DB::Context>, DB::QueryFlags, DB::QueryProcessingStage::Enum) @ 0x00000000187786cb
[2025-10-31T22:10:39.197+0000] {subprocess.py:93} INFO -   19. DB::TCPHandler::runImpl() @ 0x0000000019e2d3db
[2025-10-31T22:10:39.197+0000] {subprocess.py:93} INFO -   20. DB::TCPHandler::run() @ 0x0000000019e4f119
[2025-10-31T22:10:39.198+0000] {subprocess.py:93} INFO -   21. Poco::Net::TCPServerConnection::start() @ 0x000000001ef4ed07
[2025-10-31T22:10:39.201+0000] {subprocess.py:93} INFO -   22. Poco::Net::TCPServerDispatcher::run() @ 0x000000001ef4f199
[2025-10-31T22:10:39.203+0000] {subprocess.py:93} INFO -   23. Poco::PooledThread::run() @ 0x000000001ef15b87
[2025-10-31T22:10:39.204+0000] {subprocess.py:93} INFO -   24. Poco::ThreadImpl::runnableEntry(void*) @ 0x000000001ef13f81
[2025-10-31T22:10:39.205+0000] {subprocess.py:93} INFO -   25. ? @ 0x0000000000094ac3
[2025-10-31T22:10:39.205+0000] {subprocess.py:93} INFO -   26. ? @ 0x0000000000125a74
[2025-10-31T22:10:39.206+0000] {subprocess.py:93} INFO - 22:10:39
[2025-10-31T22:10:39.206+0000] {subprocess.py:93} INFO - 22:10:39    compiled code at target/compiled/ck_project/models/silver/silver_merchants.sql
[2025-10-31T22:10:39.207+0000] {subprocess.py:93} INFO - 22:10:39
[2025-10-31T22:10:39.207+0000] {subprocess.py:93} INFO - 22:10:39  Done. PASS=0 WARN=0 ERROR=1 SKIP=0 NO-OP=0 TOTAL=1
[2025-10-31T22:10:40.089+0000] {subprocess.py:97} INFO - Command exited with return code 1
[2025-10-31T22:10:40.105+0000] {taskinstance.py:2698} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/operators/bash.py", line 212, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 1.
[2025-10-31T22:10:40.109+0000] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=ingestion_pipeline, task_id=silver_dbt.dbt_run_merchants, execution_date=20251031T220011, start_date=20251031T221034, end_date=20251031T221040
[2025-10-31T22:10:40.125+0000] {standard_task_runner.py:107} ERROR - Failed to execute job 21 for task silver_dbt.dbt_run_merchants (Bash command failed. The command returned a non-zero exit code 1.; 1139)
[2025-10-31T22:10:40.172+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2025-10-31T22:10:40.192+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
